charts/opentelemetry-collector/values.yaml

```diff
@@ -8,6 +8,10 @@ fullnameOverride: ""
 # Valid values are "daemonset" and "deployment".
 mode: ""
 
+## If true then the user will be responsible to provide a configmap with otel-col configuration
+## So when true the config part will be ignored and the one in the volumes will be used
+useExistingConfigMap: false
```

Comment:
We could go as far as to disallow this value being true and .Values.config being anything but null.  We could add a rule to enforce this in NOTES.txt.